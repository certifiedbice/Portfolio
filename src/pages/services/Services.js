import React,{Component} from 'react';
import Context from '../../masterProvider';
import Header from '../../components/header/Header';
import Footer from '../../components/footer/Footer';
import Fade from '../../components/fade/Fade';
import Background from '../../components/background/Background';
import './Services.css';
export default class Services extends Component{
	static contextType=Context;
	componentDidMount(){
		// Compensate for content outside the viewport.
		this.context.verticalPadAndScroll();
	}
	render(){
		return(
			<>
				<Background/>
				<main>
					<section id="services">
						<div className="group">
							<div className="item">
								<h3>Art &#38; Graphic Design</h3>
								<div className="item-description">
									<p>I am available for commissioned work in the areas of:</p>
									<p>Fine Art, Illustration, Graphic Design, Tattoo, 2D Animation, 3D Modeling &#38; Animation.</p>
									<p>My technical standards are high and my work's quality is very important to me. My aim is to provide an unparalled excellence.</p>
								</div>
							</div>
							<div className="item">
								<h3>Branding Development</h3>
								<div className="item-description">
									<p>Key demographic research for a targeted strategy that is relevant to your branding needs.</p>
									<p>Identity needs are crucial to your audience understanding, identifying and connecting with your organization, not just through channels and social media but emotionally.</p>
									<p>I have experience working with a wide variety of clientele, small and large, here and there, coffee to technology.</p>
									<p>I want to give you original work that will speak for and as your brand.</p>
								</div>
							</div>
							<div className="item">
								<h3>Web Design &#38; Development</h3>
								<div className="item-description">
									<p>My goal is to provide a user &#38; device friendly website that your audience will be able to find.</p>
									<p>Valuable functionality and services like: Ecommerce, Social Media, Content Management, are a must in today's market.</p>
									<p>Whether you want to custom design, an API, a React app, a WordPress CMS, an additional feature or service added to your existing site, I have the tools necessary to provide you with results.</p>
								</div>
							</div>
						</div>
					</section>
				</main>
				<Header/>
				<Footer/>
				<Fade location={this.props.location.pathname}/>
			</>
		);
	}
}