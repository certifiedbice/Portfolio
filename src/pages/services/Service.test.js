import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom';
import Services from './Services';

it('renders without crashing',()=>{
	const pathname={location:{pathname:'/services'}};
    const div=document.createElement('div');
    ReactDOM.render(<BrowserRouter><Services location={pathname}/></BrowserRouter>,div);
});

it('renders correctly',()=>{
	const pathname={location:{pathname:'/services'}};
    const tree=renderer
        .create(<BrowserRouter><Services location={pathname}/></BrowserRouter>)
        .toJSON();
        expect(tree).toMatchSnapshot();
});