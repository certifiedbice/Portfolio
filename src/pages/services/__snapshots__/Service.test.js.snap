// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div
    id="background-image-container"
  >
    <div
      id="background-image"
    />
  </div>,
  <main>
    <section
      id="services"
    >
      <div
        className="group"
      >
        <div
          className="item"
        >
          <h3>
            Art & Graphic Design
          </h3>
          <div
            className="item-description"
          >
            <p>
              I am available for commissioned work in the areas of:
            </p>
            <p>
              Fine Art, Illustration, Graphic Design, Tattoo, 2D Animation, 3D Modeling & Animation.
            </p>
            <p>
              My technical standards are high and my work's quality is very important to me. My aim is to provide an unparalled excellence.
            </p>
          </div>
        </div>
        <div
          className="item"
        >
          <h3>
            Branding Development
          </h3>
          <div
            className="item-description"
          >
            <p>
              Key demographic research for a targeted strategy that is relevant to your branding needs.
            </p>
            <p>
              Identity needs are crucial to your audience understanding, identifying and connecting with your organization, not just through channels and social media but emotionally.
            </p>
            <p>
              I have experience working with a wide variety of clientele, small and large, here and there, coffee to technology.
            </p>
            <p>
              I want to give you original work that will speak for and as your brand.
            </p>
          </div>
        </div>
        <div
          className="item"
        >
          <h3>
            Web Design & Development
          </h3>
          <div
            className="item-description"
          >
            <p>
              My goal is to provide a user & device friendly website that your audience will be able to find.
            </p>
            <p>
              Valuable functionality and services like: Ecommerce, Social Media, Content Management, are a must in today's market.
            </p>
            <p>
              Whether you want to custom design, an API, a React app, a WordPress CMS, an additional feature or service added to your existing site, I have the tools necessary to provide you with results.
            </p>
          </div>
        </div>
      </div>
    </section>
  </main>,
  <header
    id="site-header"
  >
    <div
      id="site-info"
    >
      <h1
        id="site-title"
      >
        Portfolio of Bice
      </h1>
      <div
        id="site-description"
      >
        My various projects in all their variousness.
      </div>
      <nav>
        <ul
          id="primary-menu"
        >
          <li>
            <a
              href="/home"
            >
              Home
            </a>
          </li>
          <li>
            <a
              href="/about"
            >
              About
            </a>
          </li>
          <li>
            <a
              href="/contact"
            >
              Contact
            </a>
          </li>
          <li>
            <a
              href="/portfolio"
            >
              Portfolio
            </a>
          </li>
          <li>
            <a
              href="/resume"
            >
              Resume
            </a>
          </li>
          <li>
            <a
              href="/services"
            >
              Services
            </a>
          </li>
        </ul>
      </nav>
    </div>
  </header>,
  <footer
    id="site-footer"
  >
    Bice - 2020
    <div
      id="footer-links-container"
    >
      <a
        className="footer-social-icon-link"
        href="https://www.facebook.com/certifiedbice"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Facebook"
          className="footer-social-icons-image"
          src="facebook.png"
        />
      </a>
      <a
        className="footer-social-icon-link"
        href="https://github.com/certifiedbice"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="GitHub"
          className="footer-social-icons-image"
          src="github.png"
        />
      </a>
      <a
        className="footer-social-icon-link"
        href="https://www.instagram.com/certifiedbice"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Instagram"
          className="footer-social-icons-image"
          src="instagram.png"
        />
      </a>
      <a
        className="footer-social-icon-link"
        href="https://www.hackerrank.com/certifiedbice"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="HackerRank"
          className="footer-social-icons-image"
          src="hackerrank.png"
        />
      </a>
      <a
        className="footer-social-icon-link"
        href="https://www.linkedin.com/in/certifiedbice"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="LinkedIn"
          className="footer-social-icons-image"
          src="linkedin.png"
        />
      </a>
      <a
        className="footer-social-icon-link"
        href="https://twitter.com/certifiedbice"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Twitter"
          className="footer-social-icons-image"
          src="twitter.png"
        />
      </a>
      <a
        className="footer-social-icon-link"
        href="https://www.youtube.com/channel/UCEITFHDFDtvTIKFbUhG_E9w"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="YouTube"
          className="footer-social-icons-image"
          src="youtube.png"
        />
      </a>
    </div>
  </footer>,
  <div
    id="page-fade"
  />,
]
`;
